### PR TITLE
Building  documentation websites with Docusaurus version 2

### PR DESCRIPTION
**Pull request by calebroHQ**
Summary

In this tutorial, we have gone through how to set up Docusaurus version 2, how to customize and deploy. Docusaurus is a very simple and convenient way of deploying a documentation site or blog.

files added are index.md, hero.jpg and image1.jpg

### CODE REVIEW

**Review by cyberShaw — CHANGES_REQUESTED** [review]
Great article, @calebroHQ!
I've suggested a minor edit. Please make them and we can move to tech review.

Please add a GitHub link to the final working code

**Comment by calebroHQ** [comment]
@cyberShaw hey, it's all done.

File: articles/building-documentation-websites-with-docusaurus-v2/index.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+```
+
+### File structure
+
+The command generates a directory known as my-blog. Navigate into the directory to see the generated files. Some of the files and folders that you should pay attention to are `docusaurus.config.js`, the `src` folder, and the `blog` folder. Let us run and build the project and see how it looks like without any customization.
+
+```bash
+npm run start
+```
+
```
Comment (cyberShaw) [inline articles/building-documentation-websites-with-docusaurus-v2/index.md:50]: Please explain the difference between `npm run start` and `npm run build` in this context.

File: articles/building-documentation-websites-with-docusaurus-v2/index.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+    },
+  },
+};
+```
+
+### Modifying the style of the homepage
+
+To change the look and feel of the Homepage navigate to the `src` folder, then the `CSS` folder, and open the `custom.css` file. If you want to style your website further this is the file to add the CSS.
+
+```css
```
Comment (cyberShaw) [inline articles/building-documentation-websites-with-docusaurus-v2/index.md:132]: Please include custom css and explain how the changes reflect on the page.
the below CSS is already pre-populated in the `custom.css` file

**Review by cyberShaw — CHANGES_REQUESTED** [review]
Hey @calebroHQ. After reading the article, I found that there isn't much of features of docusaurus discussed other than what's given in the official documentation. 
https://docusaurus.io/docs

Please discuss more features like documentation, versioning, react and markdown features. this article needs more improvement with more docusaurus features discussed. Please do the revisions and we can move to the next round of review.

**Comment by hectorkambow** [comment]
@calebroHQ 

I would agree with @cyberShaw  statement - as we always want to make sure we are creating and submitting original and unique content to the community - different than what is currently on other blog posts and more importantly we never want to repeat what can be found in the official docs. 👍 

We will close this PR - just to clear some room in the PR queue - but it can still be worked on and re-submitted when it is ready for another review. 🚀

**Pull request closed without merging**